agents/mcp/test_server.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+"""
+Test script to verify the Brave source MCP server is working correctly.
+"""
```

Comment:
I'm not sure we need this, will check with @brave/build-commands-reviewers on the normal way to handle this is.